pyairtable/orm/model.py

```diff
@@ -322,6 +344,36 @@ def to_record(self, only_writable: bool = False) -> RecordDict:
         }
         ct = datetime_to_iso_str(self.created_time) if self.created_time else ""
         return {"id": self.id, "createdTime": ct, "fields": fields}
+    
+    def to_update_record(self, fields: Optional[List[str]] = None) -> UpdateRecordDict:
```

Comment:
@BAPCon same comment as above. Could be simplified by filtering the dictionary returned by `Model.to_record` with the list `fields: List[str]` of fields to update.